scripts/build

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+require('dotenv').config()
+
+const version = require('../package.json').version
+const execute = require('child_process').execSync
+
+let branch
+let enableSentry = '0'
+// Current branch
+if (process.env.NOW_GITHUB_COMMIT_REF) {
+  // ZEIT Now (no .git but an env var)
+  branch = process.env.NOW_GITHUB_COMMIT_REF
+} else if (process.env.GITHUB_REF) {
+  // Github Action
+  branch = process.env.GITHUB_REF
+} else {
+  // // Other environments
+  branch = execute(`git symbolic-ref --short -q HEAD`).toString()
```

Comment:
When we do this, we may want to double check and trim any leading or trailing whitespace.

Trying this out on a repo locally, I got `'next\n'`, which will not compare well to `development` or `master` later.

Reply:
Ah good catch!